### PR TITLE
DOC-2087: update, re-organise, and re-structure docs templates for new material

### PR DESCRIPTION
**Pull request by brian-forte**
DOC-2087: update, re-organise, and re-structure docs templates for new material

Changes:
* Added 51 files and folder outlining new material templates for
	1. new plugins
	2. new integration examples
	3. release notes

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: -new-material-templates/release-notes-template/pages/<x.y[.z]>-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+:description: Release notes for TinyMCE <x.y[.z]>
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} <x.y[.z]> was released for {enterpriseversion} and {cloudname} on <weekday>, <month> <DD>^<st|nd|rd|th>^, <YYYY>. These release notes provide an overview of the changes for {productname} 6.5, including:
```
Comment (brian-forte) [inline -new-material-templates/release-notes-template/pages/<x.y[.z]>-release-notes.adoc:12]:
```suggestion
{productname} <x.y[.z]> was released for {enterpriseversion} and {cloudname} on <weekday>, <month> <DD>^<st|nd|rd|th>^, <YYYY>. These release notes provide an overview of the changes for {productname} <x.y[.z]>, including:
```

File: -new-material-templates/release-notes-template/pages/<x.y[.z]>-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+=== <TINY-vwxyz n changelog entry>
+//#TINY-vwxyzn
+
+// CCFR here.
+
+
+[[bug-fixes]]
+== Bug fixes
+
```
Comment (brian-forte) [inline -new-material-templates/release-notes-template/pages/<x.y[.z]>-release-notes.adoc:207]:
```suggestion

{productname} <x.y[.z]> also includes the following bug fix<es>:

```

File: -new-material-templates/integration example (ie) templates/ROOT/examples/live-demos/ie-<hyphen-delimited-descriptor>/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const content_styl= `
```
Comment (TheSpyder) [inline -new-material-templates/integration example (ie) templates/ROOT/examples/live-demos/ie-<hyphen-delimited-descriptor>/index.js:1]:
```suggestion
const content_style= `
```

File: -new-material-templates/integration example (ie) templates/ROOT/examples/live-demos/ie-<hyphen-delimited-descriptor>/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    'advlist', 'autolink', 'lists', 'link', 'image', 'charmap', 'preview',
+    'anchor', 'searchreplace', 'visualblocks', 'code', 'fullscreen',
+    'insertdatetime', 'media', 'table', 'help', 'wordcount'
+  ],
+  toolbar: 'undo redo | blocks | ' +
+  'bold italic backcolor | alignleft aligncenter ' +
+  'alignright alignjustify | bullist numlist outdent indent | ' +
+  'removeformat | help',
+  content_style, // must be here if you have <css for the demo> above.
+  newdocument_content: // required — with actual material here — if your demo involves creating new documents.
```
Comment (TheSpyder) [inline -new-material-templates/integration example (ie) templates/ROOT/examples/live-demos/ie-<hyphen-delimited-descriptor>/index.js:25]:
```suggestion
  newdocument_content // required — with actual material here — if your demo involves creating new documents.
```

This is the cause of the syntax error, I'm not sure if you intended this to be a string or just a variable

**Review by TheSpyder — APPROVED** [review]

**Review by kemister85 — APPROVED** [review]
Nice work @brian-forte

**Pull request merged**